### DISCUSSION TITLE
Inclusion of SHA3

### DISCUSSION THREAD

**Discussion by tchoutri** [jedisct1/libsodium#1330] [Q&A] [answered]
@jedisct1 Is there a plan to include SHA3 in libsodium in the future? Sorry if you've mentioned it somewhere else, I could not find the information. Or should I not care about this before the 2.0.x series? :)

**Comment by jedisct1** [comment, marked as answer]
The documentation includes a [roadmap](https://doc.libsodium.org/roadmap).

SHA-3 itself, probably not. It's  slow, was quickly replaced with SHAKE, and is difficult to justify, even more with TurboSHAKE and KangarooTwelve being standardized.

We don't need yet another hash function. But a XOF, yes.

**Reply by tchoutri** [reply]
Fantastic, thank you very much!
